Modules/FT0/src/DigitQcTask.cxx

```diff
@@ -380,6 +475,19 @@ void DigitQcTask::monitorData(o2::framework::ProcessingContext& ctx)
       }
 
       setFEEmodules.insert(mChID2PMhash[chData.ChId]);
+
+      if (chData.QTCAmpl <= 0)
+        continue;
```

Comment:
Is this correct to skip channels with negative amplitudes when re-computing triggers based on data from PMs?

Reply:
this is for vertex calculations, but this condition is not enough.

Reply:
Please talk to Jacek - he had a long discussion with Dima about it and I think he can share the email with you, in which it is clarified what conditions should be used for each trigger type.

Reply:
@jotwinow

Reply:
We will need to re-iterate on this, somehow I cannot get agreement between the logic you provided and what is recorded in TCM.